strings/valentines.json

```diff
@@ -30,35 +30,37 @@
 		"The virologist is rogue, and the only cure is a kiss from you.",
 		"Would you spend some time in my upgraded sleeper?",
 		"You must be a silicon, because you've unbolted my heart.",
+		"There's no way you're rogue -- you're a fine piece of Asimov.",
 		"Are you Nar'Sie? Because there's nar-one else I sie.",
-		"If you were a taser, you'd be set to stunning.",
```

Comment:
Keep this one.

Reply:
tasers don’t stun anymore

Reply:
Exactly. That's why you should keep it.

Reply:
if I didn’t want these to correspond to in game stuff I’d just copy paste a random space pickup lines buzzfeed list

Reply:
It references the good ‘ol days, and besides, hybrid tasers still have a stun mode, even though the stunspheres don’t stun.